src/components/Calculator.js

```diff
@@ -0,0 +1,41 @@
+function Calculator() {
```

Comment:
- [ ] Please consider the first comment in this file of code as well, let's convert it to a class-based component instead of using the `function` keyword. 😄 🙏